### PR TITLE
YACReader: new recipe

### PR DESCRIPTION
**Pull request by miqlas**

### CODE REVIEW

File: media-gfx/yacreader/patches/yacreader-8.5.0.patchset [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+index 57a500d..f6b7588 100644
+--- a/YACReader/YACReader.pro
++++ b/YACReader/YACReader.pro
+@@ -68,6 +68,17 @@ LIBS         += -L/usr/lib -lpoppler-qt5
+ 
+ }
+ 
++unix:haiku{
++
++INCLUDEPATH  += /system/develop/headers/x86/poppler/qt5 /system/develop/headers/poppler/qt5
```
Comment (korli) [inline media-gfx/yacreader/patches/yacreader-8.5.0.patchset]: poppler-qt5 should provide a pkgconfig file with this information.

File: media-gfx/yacreader/patches/yacreader-8.5.0.patchset [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+diff --git a/YACReaderLibrary/server_config_dialog.cpp b/YACReaderLibrary/server_config_dialog.cpp
+index 5ce706d..169ded5 100644
+--- a/YACReaderLibrary/server_config_dialog.cpp
++++ b/YACReaderLibrary/server_config_dialog.cpp
+@@ -41,6 +41,10 @@ bool ipComparator(const QString & ip1, const QString & ip2)
+ #include <string.h> 
+ #include <arpa/inet.h>
+ 
++#ifdef Q_OS_HAIKU
++#include <bsd/ifaddrs.h>
```
Comment (korli) [inline media-gfx/yacreader/patches/yacreader-8.5.0.patchset]: should be #include <ifaddrs.h>

**Comment by korli** [comment]
Nice job!

File: media-gfx/yacreader/patches/yacreader-8.5.0.patchset (hunk 1 above)
Comment (korli) [inline media-gfx/yacreader/patches/yacreader-8.5.0.patchset]: maybe:
CONFIG += link_pkgconfig
PKGCONFIG += poppler-qt5

**Comment by miqlas** [comment]
Ok, i suppose i fexed everything. Thanks @korli!

**Pull request merged**